test/suite/test_compact09.py

```diff
@@ -75,35 +75,37 @@ def test_compact09(self):
         # Write to disk.
         self.session.checkpoint()
 
-        # Enable background compaction and exclude the two tables.
+        # Enable background compaction and exclude the two tables. Use run once to be able to 
+        # track the stats easily.
         exclude_list = f'["{self.uri_prefix}_0.wt", "{self.uri_prefix}_1.wt"]'
-        config = f'background=true,free_space_target=1MB,exclude={exclude_list}'
+        config = f'free_space_target=1MB,exclude={exclude_list},run_once=true'
```

Comment:
Was this passing 'background=true,background=true,...' before?

Reply:
Yes, but the `turn_on_bg_compaction` takes care of setting it to `true`! It was redundant.

Reply:
fair enough